bin/common.bash

```diff
@@ -376,29 +400,30 @@ validate_config() {
             sed -r 's/^.*_(..-config\.yaml): fail: (.*)/\1: \2/; / failed validation$/q' < "${schema_validation_result}"
             grep -oP '(?<=fail: )[^:]+' "${schema_validation_result}" | sort -u |
             while read -r jpath; do
-              echo -n ".$jpath = "
-              yq4 -oj ".$jpath" "${merged_config}"
+              if [[ $jpath != "(root)" ]]; then
+                echo -n ".$jpath = "
+                yq4 -oj ".$jpath" "${merged_config}"
+              fi
             done
             maybe_exit="true"
         fi
-
-        if ${maybe_exit} && ! ${CK8S_AUTO_APPROVE}; then
-            ask_abort
-        fi
     }
 
     template_file=$(mktemp --suffix="-tpl.yaml")
     append_trap "rm ${template_file}" EXIT
 
+    maybe_exit="false"
     if [[ $1 == "sc" ]]; then
         check_config "${config_template_path}/common-config.yaml" \
             "${config_template_path}/sc-config.yaml" \
             "${config_template_path}/secrets.yaml"
         yq_merge "${config_template_path}/common-config.yaml" \
             "${config_template_path}/sc-config.yaml" \
             > "${template_file}"
+        check_conditionals "${config[config_file_sc]}" "${template_file}"
         validate "${config[config_file_sc]}" "${template_file}"
         schema_validate "${config[config_file_sc]}" "${config_template_path}/schemas/config.yaml"
+        check_conditionals "${secrets[secrets_file]}" "${config_template_path}/secrets.yaml"
         validate "${secrets[secrets_file]}" "${config_template_path}/secrets.yaml"
         schema_validate "${secrets[secrets_file]}" "${config_template_path}/schemas/secrets.yaml"
     elif [[ $1 == "wc" ]]; then
```

Comment:
Nit: could you extract the common parts for sc/wc and run them after the if-statements?

Something like this?
```suggestion
    if [[ $1 == "sc" ]]; then
        check_config "${config_template_path}/common-config.yaml" \
            "${config_template_path}/sc-config.yaml" \
            "${config_template_path}/secrets.yaml"
        yq_merge "${config_template_path}/common-config.yaml" \
            "${config_template_path}/sc-config.yaml" \
            > "${template_file}"
            
        config_to_validate="${config[config_file_sc]}"
    elif [[ $1 == "wc" ]]; then
    ...
    else
    ...
    fi
    
  	check_conditionals "${config_to_validate}" "${template_file}"
    validate "${config_to_validate}" "${template_file}"
    schema_validate "${config_to_validate}" "${config_template_path}/schemas/config.yaml"
    check_conditionals "${secrets[secrets_file]}" "${config_template_path}/secrets.yaml"
    validate "${secrets[secrets_file]}" "${config_template_path}/secrets.yaml"
    schema_validate "${secrets[secrets_file]}" "${config_template_path}/schemas/secrets.yaml"
```